cfgov/jinja2/v1/the-bureau/bureau-structure/_macro-content-slider.html

```diff
@@ -0,0 +1,35 @@
+{# ==========================================================================
+
+   content-slider.render()
+
+   ==========================================================================
+
+   Description:
+
+   Renders a content slider for the given categories.
+
+   categories: An Array of categories.
+
+   ========================================================================== #}
+{% macro render(categories) -%}
+<div id="content-slider-container">
```

Comment:
Deleted in https://github.com/cfpb/cfgov-refresh/commit/3f65fc6c860c9e8fd00f10a506ea055b52a8b946.